include/msdfgl.h.in

```diff
@@ -219,6 +219,24 @@ enum msdfgl_printf_flags {
      *       to msdfgl.
      */
     MSDFGL_UTF8 = 0x08,
+
+    /**
+     * Enable newline support. The anchor point behaviour can be controlled by the flags
+     * MSDFGL_NEWLINE_ANCHOR_TOP and MSDFGL_NEWLINE_ANCHOR_BOTTOM. The newline character
+     * will move the cursor to the next line. The newline character is not rendered.
+     */
+    MSDFGL_NEWLINE = 0x10,
+
+    /**
+     * Anchor is the bottom left base point of the first glyph in the *FIRST* line.
+     * This is the default behaviour.
+     */
+    MSDFGL_NEWLINE_ANCHOR_FIRST = 0x10,
```

Comment:
Are these flags identical (both `0x10`) on purpose?

Reply:
yep. was thinking `MSDFGL_NEWLINE` to be the shorter default and then the `..._ANCHOR_FIRST` and `..._ANCHOR_LAST` to be the more detailed options. no strong feeling on it though if you'd prefer to have no aliases.

Reply:
If I were to make them aliases, I would do it by referring to the symbol, not value.

Could a better name for the flag be `MSDFGL_MULTILINE`? The anchoring could then be a completely separate setting, such as `MSDFGL_ANCHOR_TOPLEFT` etc (also leaves room for supporting for example center anchoring also in the future)

I also wonder do we need to hide the newline/multiline rendering behind a flag at all? Is there a potential usecase thst breaks with this change? And if so, maybe we can just have a flag to disable it?

Reply:
hmm... so, I've put it behind the flag not to break default behaviour, but i guess newline support does sound like a pretty good default (without it `\n`s are just rendered as spaces). for my personal use i see no need for it not to just be part of the behaviour, not even there being a flag to disable it.

as to the anchor, i was trying not to touch the function signature, hence i've added the two basic modes into the flags, but if you're ok with updating the function sig, then yeah, let's add anchor and, while at it, line spacing args. i'd suggest maybe a trick like [this](https://stackoverflow.com/a/5947763/2531987) then, to make the function call a bit more manageable?